src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @keyframes move {
+    from {
+      transform: rotate(0);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
+  &__minutes-hand {
+    background-color: #0700ff;
+    height: 20vmin;
+    width: 3vmin;
+    position: absolute;
+    bottom: 50%;
+
+    transform-origin: bottom;
+    animation: move 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    background-color: #2c8000;
+    height: 38vmin;
+    width: 1.5vmin;
+    position: absolute;
+    bottom: 50%;
+
+    animation: move 60s linear infinite;
+    transform-origin: bottom;
+  }
+
+  &__center {
+    background-color: #f6a603;
+    border-radius: 50%;
+    height: 5vmin;
+    width: 5vmin;
+    position: absolute;
+  }
+
+  &--speed-up :nth-child(1) {
+    animation: move 600s steps(60) infinite;
+  }
+
+  &--speed-up :nth-child(2) {
```

Comment:
The selector `&--speed-up :nth-child(2)` has a space before `:nth-child`, which may cause it to select any descendant rather than a direct child. Remove the space to ensure it targets the direct child element.